pkg/kubelet/kuberuntime/helpers.go

```diff
@@ -255,6 +255,46 @@ func (m *kubeGenericRuntimeManager) getSeccompProfile(annotations map[string]str
 	return ""
 }
 
+func fieldSeccompProfile(scmp *v1.SeccompProfile, profileRootPath string) *runtimeapi.SecurityProfile {
+	if scmp == nil {
+		return &runtimeapi.SecurityProfile{
+			ProfileType: runtimeapi.SecurityProfile_Unconfined,
```

Comment:
Now that I see the code assigning enum 1 unconfined to the ProfileType on nil seccomp...should unconfined be the first ordered value in the enum list?

Reply:
Intention is to move to runtime default as the default in the future.

Reply:
SGTM